### DISCUSSION TITLE
How can I help?

### DISCUSSION THREAD

**Discussion by TakshPanchal** [vedantbahel/viteED#5] [Q&A] [answered]
I am a newbie to Flutter, I know basic stuff meanwhile, I am learning by building a project and updating my knowledge. How can I help you guys in this state? currently, I am going through the code and app and then I will create some issues.

**Comment by vedantbahel** [comment, marked as answer]
at initial stages you can clone and look at what the app has. there is proposed design in the readme of root folder which you can reference. we are trying to implemented that. maybe you can design and add some widgets for the same.
lmk if this helps.
